hexchat/snapcraft.yaml

```diff
@@ -11,27 +11,26 @@ confinement: strict
 apps:
   hexchat:
     command: desktop-launch $SNAP/bin/hexchat
-    plugs: [network, network-bind, x11, unity7]
+    plugs:
+      - unity7
+      - gsettings
+      - pulseaudio
+      - network
+      - network-bind
+      - home
```

Comment:
This was less than 80 characters, same, as per our best practices, please leave current list in one line as it is for clarity.